roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -41,7 +41,7 @@ spec:
     - --advertise-address={{ ip | default(ansible_default_ipv4.address) }}
     - --etcd-servers={{ etcd_access_addresses }}
 {%   if etcd_events_cluster_enabled %}
-    - --etcd-servers-overrides=/events#{{ etcd_events_access_addresses }}
+    - --etcd-servers-overrides=/events#{{ etcd_events_access_addresses2 }}
```

Comment:
why?
 `etcd_access_addresses`  and `etcd_events_access_addresses ` 
The etcd and etcd-events clusters use different values.

Reply:
https://github.com/kubernetes/apiserver/blob/master/pkg/server/options/etcd.go#L96

```
--etcd-servers-overrides stringSlice
Per-resource etcd servers overrides, comma separated. The individual override format: group/resource#servers, where servers are URLs, semicolon separated.
```

From: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/